scipy/integrate/_quadrature.py

```diff
@@ -1304,15 +1514,15 @@ def qmc_quad(func, a, b, *, n_estimates=8, n_points=1024, qrng=None,
     >>> n_estimates = 8
     >>> res = qmc_quad(func, a, b, n_estimates=n_estimates, qrng=qrng)
     >>> res.integral, res.standard_error
-    (0.00018443143881633162, 4.709434153066518e-08)
+    (0.00018441088533413305, 1.1255608140911588e-07)
 
     A two-sided, 99% confidence interval for the integral may be estimated
     as:
 
     >>> t = stats.t(df=n_estimates-1, loc=res.integral,
     ...             scale=res.standard_error)
     >>> t.interval(0.99)
-    (0.00018426663295474533, 0.0001845962446779179)
+    (0.00018401699720722663, 0.00018480477346103947)
```

Comment:
This change doesn't seem to have anything to do with what the PR's about. If you changed this because the output given in the example is wrong, it would be best to deal with that in a separate PR.